packaging/patches/libgit2-mempack-thin-packfile.patch

```diff
@@ -0,0 +1,282 @@
+commit 9bacade4a3ef4b6b26e2c02f549eef0e9eb9eaa2
+Author: Robert Hensing <robert@roberthensing.nl>
+Date:   Sun Aug 18 20:20:36 2024 +0200
+
+    Add unoptimized git_mempack_write_thin_pack
```

Comment:
> unoptimized

After reading libgit2 again, this optimization is performed by the later call to `git_packbuilder_write_buf` instead. It is not a responsibility.

I've removed this from the PR I've submitted upstream
-  https://github.com/libgit2/libgit2/pull/6875.